src/entry/FeatureViewer.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { FC /* useRef, useEffect, useState */ } from 'react';
+import ProtvistaUniprot from 'protvista-uniprot'; // transformDataInterproAdapter, // transformDataVariationAdapter, // transformDataStructureAdapter, // transformDataProteomicsAdapter, // transformDataFeatureAdapter,
+
+import useDataApi from '../utils/useDataApi';
+import { loadWebComponent } from '../utils/utils';
+
+// const transformData = (data: any) => {
+//   const output: any = {};
+
+//   // output.STRUCTURE = transformDataStructureAdapter(data);
+
+//   return output;
+// };
+
+loadWebComponent('protvista-uniprot', ProtvistaUniprot);
+
+const FeatureViewer: FC<{ accession: string }> = ({ accession }) => {
+  // const viewerRef: any = useRef();
+  // // not false, otherwise will render as "false" in the DOM, we need undefined
+  // const [suspended, setSuspended] = useState<true | undefined>(undefined);
+  // const data = useDataApi(
+  //   `https://www.ebi.ac.uk/uniprot/api/covid-19/uniprotkb/accession/${accession}`
+  // );
+  const data = useDataApi(
+    `https://www.ebi.ac.uk/proteins/api/proteins/${accession}`
+  );
+
+  // useEffect(() => {
```

Comment:
At the moment, just let the normal component logic flow. Eventually we want to pause the logic with "suspend" attribute and inject data here.